frontend/admin/partners/editable-partner-row.component.tsx

```diff
@@ -57,7 +59,7 @@ export default function EditablePartnerRow(props: EditablePartnerRowProps) {
           headerText="Modify Partner"
           close={closeModal}
           primaryText="Modify Partner"
-          primaryAction={() => setIsSaving(true)}
+          primaryAction={() => formRef.current.requestSubmit()}
```

Comment:
This allows the phone number validation to work because it's using the form's `onSubmit`